cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/MapReduceProgramRunnerTest.java

```diff
@@ -312,9 +314,18 @@ public void testMapReduceDriverResources() throws Exception {
 
   @Test
   public void testMapreduceWithObjectStore() throws Exception {
-    final ApplicationWithPrograms app = deployApp(AppWithMapReduceUsingObjectStore.class);
+    // Deploy apps to another namespace and test cross-namespace access meanwhile
+    final ApplicationWithPrograms app = deployApp(NamespaceId.fromString("namespace:someOtherNameSpace").toId(),
```

Comment:
don't use ``Ids`` class please, its going to be removed :). ``new NamespaceId("someOtherNamespace")``.